docs/RunningFirstBenchmark.md

```diff
@@ -0,0 +1,114 @@
+# Running Your First Benchmark
+
+This tutorial is aimed at people new to the library and shows you how to run benchmarks with the Google Benchmark library.
+
+# Install the prerequisites
+
+We need the following programs:
+
+* git : to clone google benchmark and gtest libraries
+* cmake : to build google benchmark
+* make : to build google benchmark. Advanced users can use an alternative cmake backend like ninja.
+* g++ : to compile your c++ code. Advanced users can use other compilers like clang instead.
+
+On Fedora 28, the following command installs the requirements:
+
+```
+sudo dnf install --assumeyes wget git-core cmake make gcc-c++
+```
+
+At the time of writing, the versions were as follows:
+* g++ : 8.1.1 20180502
+* cmake : 3.11.2
+* gnu make: 4.2.1
+* git: 2.17.1
+
+# Build and install Google Benchmark
+
+Follow the instructions elsewhere to build and install the google-benchmark library.
+
+On Fedora 28, these commands were sufficient:
+
+```
+git clone https://github.com/google/benchmark.git
+cd benchmark
+git clone https://github.com/google/googletest.git
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=RELEASE
+make -j
+sudo make install
+```
+
+`make install` should result in lines that look like:
+
+```
+-- Install configuration: "RELEASE"
+-- Installing: /usr/local/lib/libbenchmark.a
+-- Installing: /usr/local/lib/libbenchmark_main.a
+-- Installing: /usr/local/include/benchmark
+-- Installing: /usr/local/include/benchmark/benchmark.h
+-- Installing: /usr/local/lib/cmake/benchmark/benchmarkConfig.cmake
+-- Installing: /usr/local/lib/cmake/benchmark/benchmarkConfigVersion.cmake
+-- Installing: /usr/local/lib/pkgconfig/benchmark.pc
+-- Installing: /usr/local/lib/cmake/benchmark/benchmarkTargets.cmake
+-- Installing: /usr/local/lib/cmake/benchmark/benchmarkTargets-release.cmake
+```
+
+Note the locations of the header file (`benchmark.h`) and the static library (`libbenchmark.a`); you might have to tell your compiler to look for libraries in those paths. Things worked out of the box in Fedora 28.
+
+
+# Build your first benchmark
+
+Save this file as `BenchmarkMain.cpp`:
+
+```
+#include <benchmark/benchmark.h>
+#include <string>
+
+static void BM_StringCopy(benchmark::State& state) {
+    std::string x = "hello";
+    for (auto _ : state)
+        std::string copy(x);
+}
+
+BENCHMARK(BM_StringCopy);
+BENCHMARK_MAIN();
+```
+
+Use this command to build the library:
+
+```
+g++ BenchMain.cpp -std=c++11 -lbenchmark -lpthread -O2 -o BenchMain
+```
```

Comment:
again, we work against many platforms. I think it's better to have more generic instructions about building your executable and linking against the library, then have a section below on requirements like pthread and benchmark. Users on different platforms will have different ways of linking against those libraries.